cfn-templates/cid-cfn.yml

```diff
@@ -202,6 +209,17 @@ Conditions:
         - !Equals [ !Ref DeployCostIntelligenceDashboard, "yes" ]
         - !Equals [ !Ref DeployKPIDashboard, "yes" ]
   NeedRefreshDatasets:  !Not [ !Equals  [ !Ref QuickSightDataSetRefreshSchedule, ""] ]
+  NeedDataBucketsKms: !Equals [ !Ref DataBuketsKmsKeyArns, "" ]
+  NeedDataBucketsKmsAndNeedCURTable:
+    Fn::And:
```

Comment:
You can simplify all conditions by referencing them. 
Use !Condition NeedDataBucketsKms to reference boolean value of NeedDataBucketsKms condition
See https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/conditions-section-structure.html

Reply:
Cool! This will really simplify this code! Thanks!

Reply:
done https://github.com/aws-samples/aws-cudos-framework-deployment/pull/436/commits/005e9c9c4f0532214c9de5b0bdecf61fa7dc4085